examples/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<link rel = "stylesheet" href = "styles.css">
```

Comment:
**suggestion:** <link> and <script> are usually put inside the <head> tag (and for script in some case inside the <body> tag